torch/csrc/jit/interned_strings.h

```diff
@@ -23,15 +24,17 @@ _(Sub) \
 _(Pow) \
 _(Sigmoid) \
 _(Tanh) \
+_(mul) \
```

Comment:
We should be able to remove a lot of these upper case strings now

Reply:
They still happen in ONNX, so a lot of them need to remain.

Reply:
Of course they can happen, but do we ever use symbol constants to refer to them?

Reply:
jit::kAdd still happens. I didn't bother to comb through the others.

Reply:
Not for this PR, but since the ATen ops are generated by cwrap, we should autogen their interned strings as well.